Library/ViewModels/PledgeViewModel.swift

```diff
@@ -946,8 +946,10 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
 
     // Risk Messaging Modal pledge_confirm event
     pledgeSubmitEventsSignal
+      .combineLatest(with: self.riskMessagingViewControllerDismissedProperty.signal.skipNil())
       .takeWhen(self.riskMessagingViewControllerDismissedProperty.signal)
```

Comment:
Instead of using both `combineLatest` and `takeWhen`, can you use replace both with `.takePairWhen(self.riskMessagingViewControllerDismissedProperty.signal.skipNil())`?